### DISCUSSION TITLE
"Unknown Identifier : add"

### DISCUSSION THREAD

**Discussion by SomeKindofGamer** [HaxeFlixel/flixel#2559] [Q&A]
I'm making a game in haxe
But every time i compile it says "Unknown Identifier : add"
Please help
This is the line of code which causes it:

var versionShit:FlxText = new FlxText(5, FlxG.height - 25, 0, ("Version " + ver), 12);
		versionShit.scrollFactor.set();
		versionShit.setFormat("VCR OSD Mono", 20, FlxColor.WHITE, LEFT, FlxTextBorderStyle.OUTLINE, FlxColor.BLACK);
		add(versionShit);
Is there anything i'm doing wrong?

**Comment by Geokureli** [comment]
what class are you putting this in? what class does it extend? I take it you got this code from [here](https://github.com/ninjamuffin99/Funkin/blob/faaf064e37d5a150d8aba451d740eeb81bd2e974/source/MainMenuState.hx#L99-L102). Note that this is in a class that extends FlxState, which extends FlxGroup, which defines the `add` method
